src/views/Incorporations/DefineCompany.vue

```diff
@@ -73,7 +73,7 @@
         <OfficeAddresses
           :showErrors="getShowErrors"
           :inputAddresses="addresses"
-          @update:addresses="onAddressChange($event)"
+          @update:addresses="setOfficeAddresses($event)"
```

Comment:
I just changed these to call the setters directly (and deleted the trivial methods).